docs/package-deletion-policies.md

```diff
@@ -0,0 +1,210 @@
+# Crafting a Package Deletion Policy
+
+Author: [Hayley Denbraver](https://www.github.com/hayleycd)
+Last updated: April 2025
+
+## Background
+
+In 2016 after a dispute about the naming of an open source package, a maintainer deleted all the packages he had published to npm, including a small package called `left-pad`.
```

Comment:
RE: my point about shared terminology -- it might be good to define the personas/actor names. On NuGet we have:
- site admin
- package owner (sometimes package author)
- package consumer

The term `maintainer` seems to be a little overloaded in this document. Perhaps
- registry maintainer
- package maintainer
- package consumer

But I am not sure. I guess my point goes back to well-defined roles with unambiguous names used throughout the doc.

Reply:
Looks like @hayleycd has mostly disambiguated the term 'maintainer' here. This makes me think the WG should have a general glossary, and align it's various docs around it, but that's probably out of scope for this project.

Reply:
I was thinking the same thing. a mega synonym table mapping concept across ecosystem would be so nerdy, and so cool

Reply:
Moving this to https://github.com/ossf/wg-securing-software-repos/issues/52, thanks!